### ISSUE TITLE
OS X Signature not working in some cases

### ISSUE THREAD

**Issue by feross** [feross/webtorrent-app#227]
On @mafintosh's computer, OS X 10.11.2, using the v0.1.0 DMG from the releases page, the app won't start without disabling Gatekeeper.

Any ideas?

**Comment by feross** [comment]
Fixed by updating to OS X 10.11.4. This only happened on @mafintosh's computer. Going to assume it's a one-off thing. We can re-open if we get any other reports of this.

**Comment by feross** [comment]
Just debugged this issue a whole bunch more with @jhiesey because it happened on one of his VMs that was on 10.11.2. To summarize what we learned:

- 10.11.2 does not like WebTorrent versions v0.1.0 or v0.1.1 and reports "WebTorrent Helper EH.app" and "WebTorrent Helper NP.app" as being invalidly signed.
- WebTorrent v0.2.0 works fine though, and we have no idea why. Perhaps the codesign tool on *my* computer was updated in the meantime?
- Updating to 10.11.4 (latest) fixes the issue.

Helpful commands, for future reference:

```
$ codesign --verify --deep --verbose=2 Foo.app
```

```
$ spctl -a -t exec -vv Foo.app
```

```
$ ./check-signature /Path/to/Foo.app /Path/to/Bar.pkg
```

With https://developer.apple.com/downloads/?=SignatureCheck

More documentation: https://developer.apple.com/library/mac/technotes/tn2206/_index.html
